lib/index.js

```diff
@@ -38,11 +38,16 @@ function getFrames(url, host) {
 	return frames;
 }
 
+function getTitle(url) {
+	return querystring.parse(url.split('?')[1]).title;
+}
+
 function Carousel(url, host) {
 	EventEmitter.call(this);
 	this.frames = getFrames(url, host);
 	this.carouselTimeout = null;
 	this.url = null;
+	this.title = getTitle(url);
```

Comment:
the url is passed in parameter.
the URL web api would be ideal but is experimental and also not present in node.